test/dlc_tests/container_tests/bin/testTensorFlow

```diff
@@ -34,6 +34,7 @@ set -e
 
 echo "Verify tensorflow profiler can generate logs."
 
+
```

Comment:
nit: can you remove, because I think this is causing all images to be built in the PR since it appears change to container_tests directory spawns all builds